include/crm/common/digests_internal.h

```diff
@@ -16,6 +16,14 @@
 extern "C" {
 #endif
 
+// Digest comparison results
+enum pcmk__digest_result {
+    pcmk__digest_unknown,   // No digest available for comparison
```

Comment:
ok ... that all stays strictly locally on one node meaning the actual values shouldn't matter.
having a zeroed struct default to unknown is definitely the more robust approach
but telling if the previous default to match was used or happened by error (in any way
changing behavior) isn't that trivial ... try to do my best ...

Reply:
Yep that was the only place I saw where it was allocated